src/bp/core/services/telemetry/configs.ts

```diff
@@ -0,0 +1,170 @@
+import { BotDetails } from 'botpress/sdk'
+import { BUILTIN_MODULES } from 'common/defaults'
+import LicensingService from 'common/licensing-service'
+import { buildSchema, TelemetryEvent } from 'common/telemetry'
+import { BotConfig } from 'core/config/bot.config'
+import { BotpressConfig } from 'core/config/botpress.config'
+import { ConfigProvider } from 'core/config/config-loader'
+import Database from 'core/database'
+import { calculateHash } from 'core/misc/utils'
+import { ModuleLoader } from 'core/module-loader'
+import { TelemetryRepository } from 'core/repositories/telemetry'
+import { TYPES } from 'core/types'
+import { inject, injectable } from 'inversify'
+import defaultJsonBuilder from 'json-schema-defaults'
+import _ from 'lodash'
+import ms from 'ms'
+
+import { GhostService } from '..'
+import { BotService } from '../bot-service'
+import { JobService } from '../job-service'
+import { Config } from '../module/config-reader'
+
+import { TelemetryStats } from './telemetry-stats'
+
+const DEFAULT = '***default***'
+const REDACTED = '***redacted***'
+const SECRET_KEYS = [
+  'secret',
+  'pw',
+  'password',
+  'token',
+  'key',
+  'promoted_by',
+  'description',
+  'admins',
+  'email',
+  'connection'
```

Comment:
missing `appId` (teams) and `s3` (channel-web)

Reply:
done: 233a54cfb84a4779768d539505f258b3c5a10a6a